docs/installing-ruby.md

```diff
@@ -0,0 +1,28 @@
+# Installing Ruby
+
+## Mac OS
+
+We recommend using [`ruby-install`][ruby-install] and [`chruby`][chruby] from Homebrew to manage ruby versions.
+
+1. Install [Homebrew][brew]
```

Comment:
This ordered list only has `1.` repeated!

Reply:
That is a trick that makes sure you don't have to increment the numbers so re-ordering doesnt become a pain. Once it is rendered it will be numbered properly